tests/php/Core/Injector/MethodInjectorTest/InjectableConstructorTagged.php

```diff
@@ -0,0 +1,28 @@
+<?php
+namespace SilverStripe\Core\Tests\Injector\MethodInjectorTest;
+
+use SilverStripe\Dev\TestOnly;
+
+class InjectableConstructorTagged extends InjectableConstructor implements TestOnly
+{
+    /**
+     * @var TestDependency
+     */
+    protected $protectedDependency;
+
+    /**
+     * @var array
+     */
+    protected $additionalParams;
+
+    /**
+     * @param TestDependency $protectedDependency
+     * @param array $additionalParams
+     * @Injectable
+     * @Inject(TestDependency,testTag)
```

Comment:
If this annotation is used for injector “tags” like `CacheInterface.permissions` for example, should we use dots in the annotation rather than commas? Sorry if I’m misunderstanding the use case here 😅

Reply:
Yeah this is a good point. I think my idea here when I wrote this was that it'd be similar to Doctrine annotations, which are based of classes. The dot notation makes more sense in the SS ecosystem though.